prov/efa/src/rxr/rxr.h

```diff
@@ -728,4 +730,52 @@ static inline void rxr_rm_tx_cq_check(struct rxr_ep *ep, struct util_cq *tx_cq)
 	ofi_genlock_unlock(&tx_cq->cq_lock);
 }
 
+/*
+ * @brief: check whether we should use p2p for this transaction
+ *
+ * @param[in]	ep	rxr_ep
+ * @param[in]	efa_mr	memory registration struct
+ *
+ * @return: 0 if p2p should not be used, 1 if it should, and negative FI code
+ * if the transfer should fail.
+ */
+static inline int efa_ep_use_p2p(struct rxr_ep *rxr_ep, struct efa_mr *efa_mr)
```

Comment:
function name should be `rxr_ep_use_p2p` or `efa_rdm_ep_use_p2p`

Reply:
done